src/parquet/util/rle-test.cc

```diff
@@ -318,14 +318,22 @@ TEST(BitRle, Flush) {
 // Test some random sequences.
 TEST(BitRle, Random) {
   int iters = 0;
-  while (iters < 1000) {
+
+  size_t niters = 500;
+  size_t ngroups = 1000;
+  size_t max_group_size = 16;
+  vector<int> values(ngroups + max_group_size);
+
+  while (iters < niters) {
     srand(iters++);
     if (iters % 10000 == 0) LOG(ERROR) << "Seed: " << iters;
```

Comment:
Note: I didn't write this code. Will clean it up more per the comments (and reducing runtime further)